assets/src/edit-story/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -684,4 +733,131 @@ describe('Right Click Menu integration', () => {
       });
     });
   });
+
+  describe('right click menu: background media', () => {
+    it("should duplicate the current page when clicking 'Duplicate page'", async () => {
+      const earthImage = await addEarthImage();
+      await addText();
+
+      // set image as page background
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(earthImage.id).node
+      );
+      await fixture.events.click(
+        fixture.editor.canvas.rightClickMenu.setAsPageBackground
+      );
+
+      // duplicate page
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(earthImage.id).node
+      );
+      await fixture.events.click(
+        fixture.editor.canvas.rightClickMenu.duplicatePage
+      );
+
+      // verify duplication of all content
+      const { pages } = await fixture.renderHook(() =>
+        useStory(({ state }) => ({
+          pages: state.pages,
+        }))
+      );
+
+      verifyPageDuplicated(pages);
+    });
+
+    it('should delete the current page when clicking the "Delete Page" button', async () => {
```

Comment:
Same here as above - this is the same action and we could test it at the same time as long as we add an image to the background too.